Timer.DAL/Extensions/TimerDTOExtension.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Common.ModelsDTO;
+using System;
+
+namespace Timer.DAL.Extensions
+{
+    public static class TimerDTOExtension
+    {
+        public static Timer.DAL.Entities.Timer ToTimer(this TimerDTO timerDTO)
+        {
+            Timer.DAL.Entities.Timer timer = new Timer.DAL.Entities.Timer
```

Comment:
use using alias instead

Reply:
i would be glad to but the problem is that we have a folder "Timer.DAL" and when i'm trying to add using and that create class exemplar it just says that i'm trying to use a namespace and not class
and when im trying to add using it says that such a pash doesn't contain folder entities.
So it can be fixed if we rename a Timer.DAL but i don't know if i need to do it